server/src/models/Restaurant.js

```diff
@@ -0,0 +1,20 @@
+const Model = require("./Model.js") 
+
+class Restaurant extends Model {
+    static get tableName() {
+        return "restaurants"
+    }
+
+    static get jsonSchema() {
+        return {
+            type: "object",
+            required: ["name", "location"],
+            properties: {
+                name: { type: "string" },
+                location: { type: "string" }
+            }
+        }
+    }
+}
```

Comment:
The tab size for your indentation on some of the files is 4 (like this file) and some are 2. Make sure to keep the tab size consistent. I would suggest going with 2.